src/databricks/labs/ucx/cli.py

```diff
@@ -339,6 +339,8 @@ def migrate_credentials(w: WorkspaceClient, prompts: Prompts, ctx: WorkspaceCont
     Please review the file and delete the Roles you do not want to be migrated.
     Pass aws_profile for aws.
     """
+    if len(list(w.storage_credentials.list())) > 200:
```

Comment:
Move this inside the SP migration in method `_create_storage_credentials_for_storage_accounts`